### PR TITLE
Add check for specifying CPU affinity

### PR DESCRIPTION
**Pull request by fouzhe**
* The specified CPU core should be less than 64 (winafl doesn't support more than 64 cores at the moment).
* The specified CPU core should be not allocated.

### CODE REVIEW

**Comment by ifratric** [comment]
Hmm, IIUC, this would prevent WinAfl pinning to the cpu specified by the user if another process is pinned on the same CPU, even if the other process is not very CPU-intensive. Not sure if this is desirable? Note: the current behavior is to trust the user if the user specified the exact CPU cure they want to use, and find a free core otherwise.

**Comment by fouzhe** [comment]
@ifratric Hi, thanks for your recommendation! I think you are right. I've reverted the check for the scenario that the user specifies the exact CPU core they want to use (regardless of whether another process of WinAfl is pinned on the same CPU). But I'm not sure the check, that the specified CPU core should be non-negative and less than 64, is necessary in this scenario, so I reserved this part.

File: afl-fuzz.c [hunk 1]
```diff
@@ -8036,11 +8036,14 @@ int main(int argc, char** argv) {
...
         if (cpu_aff) {
           FATAL("Multiple -c options not supported");
-        }
-        else {
-          int cpunum = 0;
+        } else {
+          u32 cpunum = 0;
+
+          if (sscanf(optarg, "%u", &cpunum) < 1 ||
+              optarg[0] == '-') FATAL("Bad syntax used for -c");
```
Comment (ifratric) [inline afl-fuzz.c:8043]: Wouldn't a safer check here be 
```
          if (sscanf(optarg, "%d", &cpunum) < 1 ||
              cpunum < 0) FATAL("Bad syntax used for -c");
```

**Comment by fouzhe** [comment]
Yeah, I think it would be better! I've updated with new patches. :)

**Comment by ifratric** [comment]
Thanks!

**Pull request merged**